samcli/lib/package/artifact_exporter.py

```diff
@@ -89,11 +89,13 @@ def do_export(self, resource_id, resource_dict, parent_dir):
             temporary_file.write(exported_template_str)
             temporary_file.flush()
 
-            url = self.uploader.upload_with_dedup(temporary_file.name, "template")
+            url = self.uploader.get(self.EXPORT_DESTINATION).upload_with_dedup(temporary_file.name, "template")
```

Comment:
I might be wrong but `self.uploader` in the `Resource` is not a dict but a specific uploader.

Reply:
Thanks for the catch, updated